.eslintrc.js

```diff
@@ -97,4 +84,32 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
   },
+  "overrides": [
+    {
+      "files": [
+        "**/test-utils/**/*.js",
+        // matching the pattern of the test runner
+        "*.test.js",
+      ],
+      env: {
+        mocha: true,
+      },
+      "rules": {
+        // does not work with wildcard imports. Mistakes will throw at runtime anyway
+        'import/named': false,
```

Comment:
didn't want to disable this globally in case this allows new bugs to be introduced. Use this opportunity to use all the mocha related rules only on test files.